lightning/src/onion_message/functional_tests.rs

```diff
@@ -138,16 +159,23 @@ impl Drop for TestCustomMessageHandler {
 impl CustomOnionMessageHandler for TestCustomMessageHandler {
 	type CustomMessage = TestCustomMessage;
 	fn handle_custom_message(&self, msg: Self::CustomMessage, responder: Option<Responder>) -> ResponseInstruction<Self::CustomMessage> {
-		match self.expected_messages.lock().unwrap().pop_front() {
-			Some(expected_msg) => assert_eq!(expected_msg, msg),
+		let add_reply_path = match self.expected_messages.lock().unwrap().pop_front() {
+			Some(Expectations { message: expected_msg, add_reply_path }) => {
+				assert_eq!(expected_msg, msg);
```

Comment:
nit: put the expectation second

Reply:
Done! [e6effd4](https://github.com/lightningdevkit/rust-lightning/commit/e6effd447d47e1c3c7771935bea58801199b60b3)